tests/integration/API/MP/APITest.php

```diff
@@ -601,6 +625,24 @@ function testItDoesNotUnsubscribeSubscriberFromWPUsersList() {
     }
   }
 
+  function testItDoesNotUnsubscribeSubscriberFromWooCommerceCustomersList() {
+    $subscriber = Subscriber::create();
+    $subscriber->hydrate(Fixtures::get('subscriber_template'));
+    $subscriber->save();
+    $segment = Segment::createOrUpdate(
+      array(
+        'name' => 'Default',
+        'type' => Segment::TYPE_WC_USERS
+      )
+    );
+    try {
+      $this->getApi()->unsubscribeFromLists($subscriber->id, array($segment->id));
+      $this->fail('WooCommerce Customers segment exception should have been thrown.');
+    } catch(\Exception $e) {
+      expect($e->getMessage())->equals("Can't subscribe to a WooCommerce Customers list with ID {$segment->id}.");
```

Comment:
Shouldn't the message here contain "unsubscribe" instead of "subscribe"?

Reply:
Fixed.